aiosmtpd/docs/handlers.rst

```diff
@@ -121,6 +121,7 @@ synchronously (i.e. they are **not** coroutines).
     a status string, such as ``'542 Internal server error'``.  If the method
     returns ``None`` or raises an exception, an exception will be logged, and a
     ``500`` code will be returned to the client.
+    Note: If connection is lost function will not be called.
```

Comment:
Please add an empty space, and strong-ify the "Note:" part, like this:

```rst
    ``500`` code will be returned to the client.

    **Note:** If connection is lost function will not be called.
```

Reply:
I prefer to have an empty line there, but I'll accept this.

Reply:
Changed.